### PR TITLE
Use hipcc.bat as a compiler on windows, instead of the perl script

### PR DESCRIPTION
**Pull request by chinmaydd**
**Summary:** Invoke "hipcc.bat" script instead of "perl hipcc". Invoking "perl hipcc" is not ideal, because there is no guarantee that hipcc is actually a perl script. Submitting in lieu of https://github.com/ROCm/Tensile/pull/2037 .

**Outcomes:** 

*What is the result of this change? What components of the project does it affect?*

At this point this is NFC - hipcc.bat script calls perl script.

**Notable changes:** N/A

**Testing and Environment:**

*What environment are you targeting (OS, ROCm version, Python versions, etc.)?* Windows

### CODE REVIEW

**Comment by chinmaydd** [comment]
ping. Could someone help us here? Thanks !

**Comment by ellosel** [comment]
Adding @TorreZuk to see if there are any unforeseeable issues with this change.

**Review by TorreZuk — APPROVED** [review]
I gave precedence to the .bat in my early commit 10ee6c6107ad21c2013f986736856b5ce8287947.  
Do we know why this was changed later (before this PR) ?

So bypassing perl and isSupportedCompiler seems fine to me as the .bat has been around for a while.  Is there no windows CI yet?  

I built with this change locally and see limited verbosity to confirm much:
This is now generated and untrue.
> UserWarning: hipcc.bat is unsupported for os nt
Local build with this change appears to pass some gemm tests.

**Comment by TorreZuk** [comment]
Okay git blame confusion, I think the original .bat was buggy so the hipcc without extension was the wrapper for perl I used, thus the use of 'perl'  + which( ).   So when you later switched to .bat taking precedence the perl should have been removed.    There is also a .exe now but without windows CI I will not suggest anything further.

**Comment by ellosel** [comment]
> Okay git blame confusion, I think the original .bat was buggy so the hipcc without extension was the wrapper for perl I used, thus the use of 'perl' + which( ). So when you later switched to .bat taking precedence the perl should have been removed. There is also a .exe now but without windows CI I will not suggest anything further.

We've been waiting on windows CI for a while... I don't recall what the reasoning was for the ordering but this changed when migrating to amdclang++.

**Comment by TorreZuk** [comment]
Well fine with this merging to unblock, but think it probably worth a new tech. debt ticket to move off hipcc entirely.  Was  the reason this was left as amdclang++ couldn't provide the hip version info?

**Comment by chinmaydd** [comment]
@TorreZuk thanks for the review. Could someone with access merge this? Thanks again !

**Comment by chinmaydd** [comment]
@ellosel thanks! If you dont mind me asking, what is the `tensile_tag.txt` update cadence in rocBLAS ?

**Pull request merged**
